src/vm/ceemain.cpp

```diff
@@ -1449,8 +1449,18 @@ void STDMETHODCALLTYPE EEShutDownHelper(BOOL fIsDllUnloading)
             {
                 if (g_IBCLogger.InstrEnabled())
                 {
-                    Thread * pThread = GetThread();
-                    ThreadLocalIBCInfo* pInfo = pThread->GetIBCInfo();
+                    Thread * pThread = GetThreadNULLOk();
+                    ThreadLocalIBCInfo* pInfo = NULL;
+
+                    if (pThread != NULL)
+                    {
+                        if (pInfo == NULL) 
```

Comment:
Yes it will be NULL

Reply:
Oops, it shouldn't be. I missed a line when copying this change from my work branch.